pkgs/top-level/all-packages.nix

```diff
@@ -1469,6 +1469,10 @@ with pkgs;
 
   mprocs = callPackage ../tools/misc/mprocs { };
 
+  mpy-utils = callPackage ../tools/misc/mpy-utils {
+    inherit (python3Packages) buildPythonApplication;
+  };
```

Comment:
Use `python3Packages.callPackage` or remove the inherit (you can get `buildPythonApplication` from `python3.pkgs`)